<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>officeHours</class>
 <widget class="QMainWindow" name="officeHours">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>OCR A Extended</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Office Hours</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/logo.png</normaloff>../Images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="officehourslabel">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Office Hours</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="allButton">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="byMonthButton">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>By month</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="monthList">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxVisibleItems">
           <number>12</number>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>January</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>February</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>March</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>April</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>May</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>June</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>July</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>August</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>September</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>October</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>November</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>December</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="yearList">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertBeforeCurrent</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="withoutRecButton">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Only without recordings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTableWidget" name="resultTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>165</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <column>
       <property name="text">
        <string>time</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>client_id</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>client_username</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>recording</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backToMenu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color:rgb(156, 120, 255);
border-radius: 15px;
color: rgb(255, 255, 255);

}

QPushButton:hover {
background-color:rgb(170, 145, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}
QPushButton:pressed {
background-color:rgb(125, 96, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}



</string>
      </property>
      <property name="text">
       <string>Back to menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
